### PR TITLE
Add brand colour for DSIT

### PR DESCRIPTION
**Pull request by beccapearce**
## What
Add brand colour for DSIT

## Why
This colour needs to be added as a new department has been set up

### CODE REVIEW

File: app/assets/stylesheets/govuk_publishing_components/components/helpers/_brand-colours.scss [hunk 1]
```diff
@@ -87,3 +87,12 @@
     border-color: govuk-colour("black");
   }
 }
+
+// new colour for DSIT for use until government-frontend has it's next deploy
```
Comment (andysellick) [inline app/assets/stylesheets/govuk_publishing_components/components/helpers/_brand-colours.scss:91]: Curious to understand this comment - what's the relationship between this and the next deployment of `government-frontend`?

Comment (beccapearce) [reply app/assets/stylesheets/govuk_publishing_components/components/helpers/_brand-colours.scss:91]: Whoops that should say govuk-frontend

Comment (andysellick) [reply app/assets/stylesheets/govuk_publishing_components/components/helpers/_brand-colours.scss]: Ah that makes sense! Thanks.

**Review by andysellick — APPROVED** [review]
👍

**Pull request merged**